ctn_waterloo/content/publications/morcos2018.bib

```diff
@@ -0,0 +1,38 @@
+@inproceedings{morcos2018,
+    author      = {Morcos, Benjamin and Stewart, Terrence C and
+                   Eliasmith, Chris and Kapre, Nachiket},
+    title       = {Implementing NEF Neural Networks on Embedded FPGAs},
+    booktitle   = {International Conference on Field-Programmable
+                   Technology (FPT)},
+    year        = {2018},
+    eventdate   = {2018-12-11/2018-12-14},
+    location    = {Naha, Okinawa, Japan},
+    publisher   = {IEEE},
+    pages       = {25-32},
+    doi         = {10.1109/FPT.2018.00015},
```

Comment:
Where did you get this doi from? This gives me not found: https://doi.org/10.1109/FPT.2018.00015 -- and googling it gives me a different paper about FPGAs.

Reply:
I got the DOI from the copy of the proceedings given to me at the conference, it hasn't been made available on the IEEE website yet. (the [2017 conference](https://ieeexplore.ieee.org/xpl/conhome.jsp?punumber=1000290) got added in Feb 2018... so not sure why it isn't up yet)

Screenshot from footer of the first page:
![screenshot30](https://user-images.githubusercontent.com/18405463/57806371-f984b100-772c-11e9-9b94-80ee1b1751d1.png)

Reply:
Resourceful! And yeah, weird that this isn't up yet.